src/plugins/foxPage/foxPage.tsx

```diff
@@ -133,9 +133,9 @@ export const FoxPage = (props: FoxPageProps) => {
             {isLargerThanMd &&
               assets.map((asset, index) => (
                 <FoxTab
+                  key={asset.assetId}
                   assetSymbol={asset.symbol}
                   assetIcon={asset.icon}
-                  isSelected={props.activeAssetId === asset.assetId}
```

Comment:
We're not passing `isSelected` here anymore so there is no reason to handle it in `<FoxTab />`.